spec/integration/belongs_to_association_spec.rb

```diff
@@ -90,35 +93,61 @@ class SimpleObject < ActiveFedora::Base
         belongs_to :complex_collection, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf, class_name: 'ComplexCollection'
       end
 
-      class ComplexObject < SimpleObject
+      #NOTE: As Active Fedora does alphabetical ordering of RDF datastreams automatically, the "Z" is to insure this is stored
```

Comment:
Yea, I'm just picking nits about the precision of the statement. How about saying that "RDF assertions seem to be returned in alphabetical order..."